package.json

```
@@ -98,13 +98,13 @@
     "test-e2e": "tools/postgres/dbtool.js prepare-test-db && tools/test.sh test-e2e --timeout=60s --verbose",
     "test-screenshot": "tools/test.sh test-screenshot --timeout=5m",
     "test-wkorg-screenshot": "tools/test.sh test-wkorg-screenshot --timeout=5m",
-    "refresh-screenshots": "rm -rf frontend/javascripts/test/screenshots/** && docker compose up screenshot-tests",
+    "refresh-screenshots": "rm -rf frontend/javascripts/test/screenshots/** && ./tools/docker-env.sh bash -c 'for i in {1..3}; do yarn test-screenshot && break; done'",
```

I am getting a `No matches found: "frontend/javascripts/test/screenshots/*` error when trying to execute the script while the screenshots are already removed. There might be multiple possible reasons for this maybe zsh behaviour or so. This can be fixed by forcing execution in bash:

```suggestion
    "refresh-screenshots": "bash -c 'rm -rf frontend/javascripts/test/screenshots/**' && ./tools/docker-env.sh bash -c 'for i in {1..3}; do yarn test-screenshot && break; done'",
```

Could you please fix that? :pray:

---

Moreover, the screenshot tests do not work for me :/. Don't know why but the dataset id cannot be retrieved successfully :/
>   dtype_datasets_rendering.screenshot › Dataset IDs were retrieved successfully

>  public-test/test-bundle/test/puppeteer/dtype_datasets_rendering.screenshot.js:158

>  157: import_dataset_rendering_helpers.test.serial("Dataset IDs were retrieved successfully", (t) => {
>   158:   (0, import_dataset_rendering_helpers.assertDatasetIds)(t, datasetNames, datasetNameToId);     
>   159: });                                                                                             

>  Dataset ID not found for "dtype_test_uint8_color"

>  Value is not truthy:

>  undefined

@philippotto Do you maybe know more about this? I checked my browser stack credentials and wk auth token env var and they seemed to be alright.